runelite-client/src/main/java/net/runelite/client/plugins/discord/DiscordState.java

```diff
@@ -103,15 +103,18 @@ else if (changed)
 		lastAction = Instant.now();
 		final DiscordGameEventType newEvent = lastQueue.get(lastQueue.size() - 1);
 
-		if (lastEvent != newEvent)
+		if (event != newEvent)
 		{
-			lastEvent = newEvent;
+			event = newEvent;
 
-			lastPresence = DiscordPresence.builder()
-				.state(lastEvent.getState())
-				.details(lastEvent.getDetails())
-				.startTimestamp(startOfAction)
-				.build();
+			final DiscordPresence.DiscordPresenceBuilder discordPresenceBuilder = DiscordPresence.builder()
+				.state(event.getState())
+				.details(event.getDetails())
+				.startTimestamp(startOfAction);
```

Comment:
Are you sure this line should be here?

Reply:
Oh... well, that kind of defeats purpose of the PR, oops